submissions/description/cross_site_scripting_xss/flash_based/template.md

```diff
@@ -0,0 +1,30 @@
+# Flash-Based Cross-Site Scripting
+
+## Overview of the Vulnerability
+
+Cross-Site Scripting (XSS) is a type of injection attack where malicious JavaScript is injected into a website. When a user visits the affected web page, the Javascript executes within that user’s browser in the context of the domain. Flash-based XSS can be found on this domain which allows an attacker to control code that is executed within a user’s browser.
```

Comment:
I think clarifying what a Flash based XSS is compared to a standard XSS is here will help clarify it for a researcher, and the customer.